tests/iio_stresstest.c

```diff
@@ -388,18 +398,37 @@ static void *client_thread(void *data)
 	return (void *)EXIT_FAILURE;
 }
 
-static unsigned long int clamp(const char *name, unsigned long int val,
+static unsigned long int sanitize_clamp(const char *name, const char *val,
```

Comment:
if we rename `const char *val` -> `const char *argv`, we can rename `tmp` -> `val` as a local variable;
that would reduce some bits of patch in the clamp() part;

Reply:
done